build-info-extractor/src/main/java/org/jfrog/build/extractor/executor/CommandExecutor.java

```diff
@@ -154,8 +180,7 @@ public CommandResults exeCommand(File execDir, List<String> args, List<String> c
     private CommandResults getCommandResults(boolean terminatedProperly, List<String> args, String output, String error, int exitValue) {
         CommandResults commandRes = new CommandResults();
         if (!terminatedProperly) {
-            error += System.lineSeparator() + format("Process '%s' had been terminated forcibly after timeout of %d seconds.",
-                    join(" ", args), TIMEOUT_SECONDS);
+            error += System.lineSeparator() + format("Process '%s' had been terminated forcibly after timeout of %d minutes.", join(" ", args), EXECUTION_TIMEOUT_MINUTES);
```

Comment:
If the timeout is overridden by a parameter, then this log message will be inaccurate. It'd be best to use the timeout parameter instead of the const here.